src/test/java/org/apache/bcel/CounterVisitorTestCase.java

```diff
@@ -217,4 +217,8 @@ public void testSyntheticCount() {
     public void testUnknownCount() {
         assertEquals(0, getVisitor().unknownCount, "unknownCount");
     }
+    @Test
```

Comment:
Please add a blank line to separate methods (watch out for trailing whitespace).

Reply:
fixed in https://github.com/apache/commons-bcel/pull/290/commits/ed41dba669d77ce3a7fca96471691eef4af9a79d